recipes-kernel/linux/linux-boundary_4.14.x.bb

```diff
@@ -0,0 +1,30 @@
+# Adapted from linux-imx.inc, copyright (C) 2013, 2014 O.S. Systems Software LTDA
+# Released under the MIT license (see COPYING.MIT for the terms)
+
+require recipes-kernel/linux/linux-imx.inc
+
+SUMMARY = "Linux kernel for Boundary Devices boards"
+
+SRC_URI = "git://github.com/boundarydevices/linux-imx6.git;branch=${SRCBRANCH} \
+"
+
+LOCALVERSION = "-2.0.0-ga+yocto"
+SRCBRANCH = "boundary-imx_4.14.x_2.0.0_ga"
+SRCREV = "9abee5da4fbc750da202dbba086ff355c0655d54"
+DEPENDS += "lzop-native bc-native"
+COMPATIBLE_MACHINE = "(nitrogen6x|nitrogen6x-lite|nitrogen6sx|nitrogen7|nitrogen8m|nitrogen8mm)"
+
+KERNEL_DEFCONFIG ?= "boundary_defconfig"
+
+S = "${WORKDIR}/git"
+
+#Copy boundary_defconfig from git
+do_copy_default_config() {
+    if [ ! -f  ${WORKDIR}/defconfig ]; then
+        if [ -f ${S}/arch/${ARCH}/configs/${KERNEL_DEFCONFIG} ]; then
+            cp ${S}/arch/${ARCH}/configs/${KERNEL_DEFCONFIG} ${WORKDIR}/defconfig
+        fi
```

Comment:
What happens if we the `KERNEL_DEFCONFIG` has a typo?

Reply:
We have edited this to bberror out if invalid KERNEL_DEFCONFIG set.